src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionService.java

```diff
@@ -150,12 +150,14 @@ public ProgrammingSubmission processNewProgrammingSubmission(ProgrammingExercise
 
             participationService.resumeProgrammingExercise(programmingExerciseStudentParticipation);
             // Note: in this case we do not need an empty commit: when we trigger the build manually (below), subsequent commits will work correctly
-            try {
-                continuousIntegrationTriggerService.orElseThrow().triggerBuild(participation);
-            }
-            catch (ContinuousIntegrationException ex) {
-                // TODO: This case is currently not handled. The correct handling would be creating the submission and informing the user that the build trigger failed.
-            }
+        }
+
+        // TODO: there might be cases in which Artemis should NOT trigger the build
```

Comment:
Which ones? Can you list them?

Reply:
No idea, TODO added by Stephan. I assume just something to look out for in the future.